pkg/loader/githubloader.go

```diff
@@ -74,8 +74,15 @@ func newGithubLoader(repoUrl string, fs fs.FileSystem) (*githubLoader, error) {
 }
 
 // isRepoUrl checks if a string is a repo Url
-func isRepoUrl(s string) bool {
-	return strings.Contains(s, ".com") || strings.Contains(s, ".org") || strings.Contains(s, "https://")
+func isRepoUrl(s string, fs fs.FileSystem) bool {
```

Comment:
Can you move the change into another function like `isLocalTarget` and leave `isRepoUrl` unchanged?